mergecounts.wdl

```diff
@@ -3,29 +3,38 @@ task MergeCounts {
 
     Array[File] inputFiles
     String outputFile
-    String idVar
-    String measurementVar
+    Int featureColumn
+    Int valueColumn
+    Boolean inputHasHeader
 
     # Based on a script by Szymon Kielbasa/Ioannis Moustakas
     command <<<
         set -e -o pipefail
+        mkdir -p ${sub(outputFile, basename(outputFile) + "$", "")}
         ${preCommand}
-        R --no-save --slave <<CODE > ${outputFile}
+        R --no-save <<CODE
```

Comment:
This merges count tables right? Can't we use python for this? (Scala? Maybe a lot of overhead...). But I guess it does not matter as long as it works.

Reply:
I think at some point I asked if there was a Biopet tool for this and I was told to just use the R script from Szymon.